### PR TITLE
feat(client): add --token argument for swcli instance login command

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
image

## Modules
- [x] Client

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/core/instance/view.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def login(self, instance: str, username: str, password: str, alias: str) -> None
         r = requests.post(
             url,
             timeout=DEFAULT_HTTP_TIMEOUT,
-            data={"userName": username, "userPwd": password},
+            data=auth_request,
```
Comment (jialeicui) [inline client/starwhale/core/instance/view.py:46]: My understanding is that login API is used to generate tokens, there is no need to send the token to login API
cc @dreamlandliu

File: client/starwhale/core/instance/cli.py [hunk 2]
```diff
@@ -28,15 +29,25 @@ def _select(instance: str) -> None:
 
 @instance_cmd.command("login")
 @click.argument("instance", default="")
-@click.option("--username", prompt="username", required=True)
-@click.password_option(confirmation_prompt=False)
+@click.option("--username")
+@click.option("--password")
+@click.option("--token", help="Get token from Starwhale Cloud page")
```
Comment (jialeicui) [inline client/starwhale/core/instance/cli.py:34]: Would it have been better if the word `Get` was removed?

**Comment by jialeicui** [comment]
LGTM

File: client/starwhale/core/instance/cli.py (hunk 2 above)
Comment (tianweidut) [reply client/starwhale/core/instance/cli.py:34]: ok

File: client/starwhale/core/instance/view.py (hunk 1 above)
Comment (tianweidut) [reply client/starwhale/core/instance/view.py:46]: I hope someone API can check the token validity and get the user info.

Comment (tianweidut) [reply client/starwhale/core/instance/view.py:46]: Maybe using `/api/v1/user/current` API to check is the better choice.

Comment (tianweidut) [reply client/starwhale/core/instance/view.py]: @jialeicui  updated

**Review by jialeicui — APPROVED** [review]

**Pull request merged**